addons/mail/static/src/js/activity.js

```diff
@@ -740,9 +742,23 @@ var KanbanActivity = BasicActivity.extend({
     },
 });
 
+// -----------------------------------------------------------------------------
+// Activity Exception Widget to display Exception icon ('activity_exception' widget)
+// -----------------------------------------------------------------------------
+var ActivityException = basic_fields.FieldChar.extend({
+
+    _renderReadonly: function () {
```

Comment:
so in edit the field is just an input? Don't you always want to render it in readonly?